### ISSUE TITLE
SQS to Lambda - the incoming event source ARN does not contain the correct region

### ISSUE THREAD

**Issue by deganmee** [localstack/localstack#1763]
## SQS to Lambda - the source_arn does not contain the correct region for the source

An SQS queue and lambda function are provisioned in us-west-2 regiona. An event source mapping exists to enable SQS to Lambda triggers.  When a message arrives in the SQS queue, localstack compares the source_arn for the queue with the ARN in the EventSourceMapping (if they match, then the lambda is invoked).  However, the source ARN for the queue incorrectly contains `us-east-1` region, and so does not match the EventSourceMapping ARN, which correctly contains `us-west-1`

### SQS Queue ARN contains us-west-2:
```
{
    "Attributes": {
        "QueueArn": "arn:aws:sqs:us-west-2:000000000000:local-queue"
    }
}
```

### Lambda function ARN contains us-west-2:
```
{
    "Functions": [
        {
            "FunctionName": "local-lambda",
            "FunctionArn": "arn:aws:lambda:us-west-2:000000000000:function:local-lambda",
        }
    ]
}
```

### EventSourceMapping refers to the correct ARNs:
```
{
    "EventSourceMappings": [
        {
            "EventSourceArn": "arn:aws:sqs:us-west-2:000000000000:local-queue",
            "FunctionArn": "arn:aws:lambda:us-west-2:000000000000:function:local-lambda",
        }
    ]
}
```

### When an SQS message is received in local-queue, the source ARN is incorrect:
```
Found 0 source mappings for event from SQS queue arn:aws:sqs:us-east-1:000000000000:local-queue: []
```

### Additional detail

Sending a message using Java client with region us-west-2, source_arn contains us-east-1

When sending a message using AWS CLI, changing the --region will also change the region in the source_arn.

### Expected behavior
The region in the source_arn should match the region that the resource was created in, OR, the event source mapping should not expect region to match

Relevant for event source mapping: get_event_sources in localstack/services/awslambda/lambda_api.py

**Comment by whummer** [comment]
Thanks for reporting @deganmee . This should be fixed in #1999 - can you please give it a try with the `latest` version of the Docker image? Please re-open this issue if this problem persists. Thanks

**Comment by ciCciC** [comment]
Please correct me if Im wrong. Im still having this issue, even with the latest version.
